nservicebus/messaging/routing.md

```diff
@@ -58,6 +58,7 @@ Events can be received by multiple logical endpoints, however even in case of sc
 
 [Multicast transports](/nservicebus/transports/#types-of-transports-multicast-enabled-transports) support Publish-Subscribe pattern natively. In this case the subscriber uses the APIs of the transport to create a route for a given subscribed message type.
 
+Note: Azure Service Bus `EndpontOrientedTopology` requires [publishers names](/nservicebus/azure-service-bus/publisher-names-configuration.md) to be configured.
```

Comment:
assuming we're talking about names of multiple publishers and not multiple names of a single publisher seems to be correct. if not then please fix directly on master @SeanFeldman